types/clone/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for clone 0.1.11
+// Type definitions for clone 2.1.2
 // Project: https://github.com/pvorb/node-clone
-// Definitions by: Kieran Simpson <https://github.com/kierans/DefinitelyTyped>
+// Definitions by: DG-za <https://github.com/DG-za/DefinitelyTyped>
```

Comment:
So this seems wrong,
I think the original owner line should be left and look like:
```suggestion
// Definitions by: Kieran Simpson <https://github.com/kierans>
```
you should add your line, but change it to:
```ts
DG-za <https://github.com/DG-za>
```

Reply:
Fixed and added my line as a second owner.